src/main/java/cn/nukkit/scoreboard/data/DisplaySlot.java

```diff
@@ -1,16 +1,11 @@
-package cn.nukkit.network.protocol.types;
```

Comment:
I think we can leave this part in network.protocol package

Reply:
But in pnx it is in the package cn.nukkit.scoreboard

Reply:
It doesn’t need to be exactly the same as pnx